docs/en/deployment/hadoop_java_sdk.md

```diff
@@ -21,7 +21,7 @@ JuiceFS use local mapping of `user` and `UID`. So, you should [sync all the need
 
 You should first create at least one JuiceFS file system to provide storage for components related to the Hadoop ecosystem through the JuiceFS Java SDK. When deploying the Java SDK, specify the metadata engine address of the created file system in the configuration file.
 
-To create a file system, please refer to [JuiceFS Quick Start Guide](../getting-started/quick_start_guide.md).
+To create a file system, please refer to [JuiceFS Quick Start Guide](/community/quick_start_guide).
```

Comment:
Can we use relative path here?

Reply:
Sure, that's a mistake, I have fixed it.